### ISSUE TITLE
Request Frederik Barkhof to add to PDF report

### ISSUE THREAD

**Issue by HenkMutsaerts** [ExploreASL/ExploreASL#224]
Frederik asked me to add minimal identification information to our PDF report, about the scanner and ASL protocol used, with some parameters. We can easily do this, but it would be tricky perhaps to still fit this on a single PDF page :)

### Release note
Nothing - moved to another issue.

**Comment by MichaelStritt** [comment]
Hey, I had some time this morning and I wanted to fix some of these things anyway.

I checked out the options for **multi page PDFs** in Matlab. The only open-source option is to use [append_pdf](https://de.mathworks.com/matlabcentral/fileexchange/31215-append_pdfs), which sadly uses ghostscript, which can't be used commercially. Then there's also the nice solution to use a [Matlab Reporter](https://de.mathworks.com/help/rptgen/ug/what-is-a-reporter.html), which sadly requires an extra license and most of the functionality was introduced in 2019 or later.

So I guess we should stick to **single page PDFs**.

Why do we use a different font size on Windows compared to Unix?

I committed some initial ideas, maybe they help you @BeatrizPadrela.
